tests/goth_tests/test_webapp.py

```diff
@@ -0,0 +1,75 @@
+"""Test Simple, db-enabled dApp."""
```

Comment:
hmm... I thought the task only entailed adding a test for the simple service? ....

Reply:
From task description: `add test case which will run dapp consistent of at least two nodes with depends_on dependency`